packages/react-liveness/src/components/FaceLivenessDetector/service/utils/streamProvider.ts

```diff
@@ -117,6 +118,7 @@ export class LivenessStreamProvider extends AmazonAIInterpretPredictionsProvider
       credentials,
       region: this.region,
       customUserAgent: getAmplifyUserAgent(),
+      requestHandler: new WebSocketFetchHandler({ connectionTimeout: 10_000 }),
```

Comment:
What should the value of timeout here be? Currently set to 10s but this is probably too high

Reply:
10 kinda makes sense though base on the number you mentioned in the description, plus high timeout does not cause side effects for users on fast network but it allows more wait time for users on slower network?

Reply:
that's correct, mostly affects slower network users